src/App.tsx

```diff
@@ -1,9 +1,29 @@
-import React from 'react';
+import React, { ChangeEventHandler, useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+function filteredMovies(query: string) {
+  if (!query) {
+    return moviesFromServer;
+  }
+
+  const lowerCaseQuery = query.toLocaleLowerCase();
+
+  return moviesFromServer.filter(
+    movie =>
+      movie.title.toLocaleLowerCase().includes(lowerCaseQuery) ||
+      movie.description.toLocaleLowerCase().includes(lowerCaseQuery),
+  );
+}
+
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const handleSearch: ChangeEventHandler<HTMLInputElement> = e => {
+    setQuery(e.target.value.trim());
+  };
```

Comment:
Don't use one-letter variable naming. (It could be confusing for another developer, who reads your code. Also, it would really hard to find this variable in the code editor using search) [[checklist]](https://github.com/mate-academy/react_movies-list-filter/blob/master/checklist.md)